lakefs_provider/hooks/lakefs_hook.py

```diff
@@ -31,6 +33,7 @@ class LakeFSHook(BaseHook):
     :type lakefs_conn_id: str
     """
     conn_name_attr = "lakefs_conn_id"
+    client_id = f"lakefs-airflow-provider/{__version__ if __version__ else 'dev'}"
```

Comment:
version will be part of the package, no need for a fallback a linter should find it too.